src/main/java/io/vertx/mqtt/impl/MqttClientImpl.java

```diff
@@ -82,6 +83,7 @@
   private static final int MIN_TOPIC_LEN = 1;
   private static final String PROTOCOL_NAME = "MQTT";
   private static final int PROTOCOL_VERSION = 4;
+  private static final int DEFAULT_INTERNAL_NET_CLIENT_IDLE_TIMEOUT = 0;
```

Comment:
I think that `DEFAULT_IDLE_TIMEOUT` is enough

Reply:
Agree